src/__tests__/operate.test.jsx

```diff
@@ -0,0 +1,23 @@
+import operate from '../Logic/operate';
```

Comment:
- [ ]  Your test suites are failing, the first error is from your import statement. The 'L' in login is in lowercase. Kindly correct this please so your tests can pass.